### DISCUSSION TITLE
woowcommerce barcode , QR code request

### DISCUSSION THREAD

**Discussion by souqalshoorja** [inspireui/support#5365] [Working on Features]
Hi we need to display barcode , QR code in woocommerce Products pages , Orders...(  FluxStore Multi Vendors )
This addition facilitates the process of buying and selling, accounts, stores, and what is available in stock... We hope to discuss this topic because it brings great benefit to store owners in direct selling and through the application

souqalshoorja.com

**Comment by Haniel5** [comment]
Hi, this feature is available on version 2.5.3 on FluxStore WooCommerce, FluxStore Multi Vendors, and FluxStore Pro (woocommerce) as our video https://youtu.be/yC-Z3hZ6hu4 - Very Best!
